pygmt/src/grdclip.py

```diff
@@ -0,0 +1,91 @@
+"""
+grdclip - Change the range and extremes of grid values.
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    R="region",
+    Sa="above",
+    Sb="below",
+    Si="interval",
+    Sr="old",
+    V="verbose",
+)
+@kwargs_to_strings(
+    R="sequence",
+    Sa="sequence",
+    Sb="sequence",
+    Si="sequence",
+    Sr="sequence",
+)
+def grdclip(grid, **kwargs):
+    r"""
+    Sets values in a grid that meet certain criteria to a new value.
+
+    Produce a clipped ``outgrid`` or xarray.Data Array version of the
+    input ``grid`` file.
+
+    The parameters ``above`` and ``below`` allow for a given value to be set
+    for values above or below a set amount, respectively. This allows for
+    extreme values in a grid, such as points below a certain depth when
+    plotting Earth relief, to all be set to the same value.
+
+    Full option list at :gmt-docs:`grdclip.html`
+
+    {aliases}
+
+    Parameters
+    ----------
+    grid : str or xarray.DataArray
+        The file name of the input grid or the grid loaded as a DataArray.
+    outgrid : str or None
+        The name of the output netCDF file with extension .nc to store the grid
+        in.
+    {R}
+    above : str or list or tuple
+        [*high*, *above*].
+        Set all data[i] > *high* to *above*.
+    below : str or list or tuple
+        [*low*, *below*].
+        Set all data[i] < *low* to *below*.
+    interval : str or list or tuple
+        [*low*, *high*, *between*].
```

Comment:
Is *between* better than *interval*, because we use *above* for *high/above*.
```suggestion
    between : str or list or tuple
        [*low*, *high*, *between*].
```